### PR TITLE
Tidy CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by apachano**
Authors:
@apachano 

## Summary of changes
-  Cleans up some of the language around guidelines

## On approval
Please merge
*Let's get two approvals for this one before merging

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -18,32 +20,32 @@ Familiarity with Git, specifically with using branches, is a must for interactin
...
-
 ### PR checklist
 This section lists the requirements for merging code. All of these requirements must be satisfied before code is merged into any branch.
 
 All Branches:
 - Accomplish the feature(s) it was designed to accomplish
 - Has the branch it's merging into merged onto itself and all conflicts are resolved
 - Clean of all binaries and other non-source material
-- Code is documented with doxygen style comments
+- All classes, public functions, and public variables are documented with doxygen style comments
```
Comment (beeperdeeper089) [inline CONTRIBUTING.md:48]: Drop the public, any function in general should really be documented for ease of later use.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -135,15 +137,16 @@ class FooBar {
...
+	 * @return What does the value returned by this function mean?
 	 */
   	int doThing(std::string desc);
   	
   	float randomNumber; //< This is a single line description for a member
 
-      /// @brief This member variable requires a longer single line comment. (filling space)
-      bool foobarred;
+  	/// @brief This member variable requires a longer single line comment. (filling space)
+    bool  foobarred;
```
Comment (beeperdeeper089) [inline CONTRIBUTING.md:149]: indentation borked?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 # GENTEN STUDIOS CONTRIBITING GUIDELINES
-These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request. However a PR may have changes requested on it if it does not adhear to guidelines outlined here.
+
+If you are just browsing through quick before starting, its worth keeping [the PR checklist](#pr-checklist) in mind before you open a new PR to help make your PR review go smoother.
```
Comment (tobyplowy) [inline CONTRIBUTING.md:4]: On line 4 shouldn't the "it's" not "its"?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -18,32 +20,32 @@ Familiarity with Git, specifically with using branches, is a must for interactin
 **The Master Branch** is the *most* stable source for the code, this contains major/ minor releases and nothing else. Only release and hotfix branches can be pushed here.
 - All Genten Developers and Staff Members must approve any push to the master branch - unless said otherwise (e.g. last minute hotfixes).
 - The code must be thoroughly tested on all supported systems before being pushed here.
+- Pre-compiled distributable code should be pushed to the releases section of github when pushing code to the Master branch.
```
Comment (tobyplowy) [inline CONTRIBUTING.md:23]: I'm a bit OCD on this one but can you type "github" as "GitHub" not a must but just looks better. (If you do it do it everywhere)

**Pull request merged**